exercises/say/say_test.rb

```diff
@@ -1,128 +1,116 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'say'
 
+# Test data version: e3bd4a2
 class SayTest < Minitest::Test
-  def test_0
-    assert_equal 'zero', Say.new(0).in_english
+  def test_zero
+    # skip
+    question = '0'
+    assert_equal(zero, SayProblem.new(question).answer)
```

Comment:
It looks like the generated tests aren't using a string for an assertion. `zero` rather than `'zero'`